stylesheets/bootstrap.scss

```diff
@@ -72,4 +72,13 @@
 // Custom breadcrumb separator
 .breadcrumb > li + li::before {
     content: ">"
-}
\ No newline at end of file
+}
+
+pre {
+    overflow-x: auto;
+}
+
+pre code {
+    overflow-wrap: normal;
```

Comment:
It looks like [browser support](https://www.w3schools.com/cssref/css3_browsersupport.asp) for this has been around for quite a while. I can add `word-wrap: normal` too just to be safe, but I don't think we need it at this point in time.